packages/babel-plugin-proposal-private-methods/test/fixtures/static-accessors/tagged-template/exe.js

```diff
@@ -0,0 +1,11 @@
+class Foo {
```

Comment:
I don't think we need tests for accessors, since tagged templates are specifically about the call behavior and not about how the tag is obtained.